src/main/java/net/starschema/clouddb/jdbc/BQConnection.java

```diff
@@ -304,6 +309,16 @@ private static Integer parseIntQueryParam(String param, @Nullable String paramVa
     return val;
   }
 
+  /**
+   * Return an empty list if {@code string} is null. Otherwise, return an array of strings iff
+   * {@code string} can be parsed as an array when split by {@code delimiter}.
+   */
+  private static List<String> parseArrayQueryParam(@Nullable String string, Character delimiter) {
+    return string == null
+        ? Collections.emptyList()
+        : Arrays.asList(string.split(delimiter + "\\s*"));
```

Comment:
What's the trailing `\\s*` for?  Handling a comma at the end of the list or something?

Reply:
Any potential whitespace between SA emails -- alternatively we could trim each element after the split. Or, just trust that we'll get a `targetServiceAccount` sans whitespace

For things like this that might sneak in: `<sa-1>@services.com,  <sa-2>@services.com`

Reply:
Ah, that makes sense.  Probably good enough for now.  I think split + strip would be more clear, except that this is java, so it might not actually be that clear, because there's no simple `map` functionality.